src/program.ts

```diff
@@ -35,21 +35,33 @@ const packageJSON = require('../package.json');
 program
     .version('Version ' + packageJSON.version)
     .name(packageJSON.name)
+    .option('--browser <browser>', 'Browser or chrome channel to use, possible values: chrome, firefox, webkit, msedge.')
+    .option('--cdp-endpoint <endpoint>', 'CDP endpoint to connect to.')
+    .option('--executable-path <path>', 'Path to the browser executable.')
     .option('--headless', 'Run browser in headless mode, headed by default')
+    .option('--port <port>', 'Port to listen on for SSE transport.')
     .option('--user-data-dir <path>', 'Path to the user data directory')
     .option('--vision', 'Run server that uses screenshots (Aria snapshots are used by default)')
-    .option('--port <port>', 'Port to listen on for SSE transport.')
-    .option('--cdp-endpoint <endpoint>', 'CDP endpoint to connect to.')
-    .option('--channel <channel>', 'Channel to use for browser, possible values: chrome, msedge, chromium. Default: chrome')
-    .option('--executable-path <path>', 'Path to the browser executable.')
     .action(async options => {
+      let browserName: 'chromium' | 'firefox' | 'webkit' = 'chromium';
```

Comment:
I think it's much easier to understand and maintain a big switch over `options.browser` that assigns both `browserName` and `channel` for each value.